packages/satusehat/configuration-schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "baseUrl": {
+      "title": "Base URL",
```

Comment:
Will `baseUrl` ever be different in satusehat? Can users self-deploy or does everyone use the same satusehat domain?

At the  very least, baseUrl I think should be optional (it is currently required), defaulting to satusehat.com or whatever. But I suspect we want to remove it entirely.

Reply:
Ok, there are at least two satusehat APIs - the main one and the sandbox. So we do need baseURL.

Let's leave it as required for the time being, to ensure that users intentioanlly set the right server